modules/web/src/assets/css/global/_main.scss

```diff
@@ -533,6 +533,13 @@ km-dashboard {
       overflow-y: auto;
       padding-right: 24px;
     }
+
+    // Override material default "Forms" field padding
+    & km-preset-settings-step {
```

Comment:
We're using `fxLayoutGap` property on `form` inside edit provider components for each provider so we should reduce that value instead.

```
      <km-aks-settings *ngSwitchCase="Providers.AKS"
                       [formControl]="form.get(Controls.Settings)">
      </km-aks-settings>
      <km-alibaba-settings *ngSwitchCase="Providers.ALIBABA"
                           [formControl]="form.get(Controls.Settings)">
      </km-alibaba-settings>
      <km-anexia-settings *ngSwitchCase="Providers.ANEXIA"
                          [formControl]="form.get(Controls.Settings)">
      </km-anexia-settings>
      <km-aws-settings *ngSwitchCase="Providers.AWS"
                       [formControl]="form.get(Controls.Settings)">
      </km-aws-settings>
      <km-azure-settings *ngSwitchCase="Providers.AZURE"
                         [formControl]="form.get(Controls.Settings)">
      </km-azure-settings>
      <km-digitalocean-settings *ngSwitchCase="Providers.DIGITALOCEAN"
                                [formControl]="form.get(Controls.Settings)">
      </km-digitalocean-settings>
      <km-eks-settings *ngSwitchCase="Providers.EKS"
                       [formControl]="form.get(Controls.Settings)">
      </km-eks-settings>
      <km-gcp-settings *ngSwitchCase="Providers.GCP"
                       [formControl]="form.get(Controls.Settings)">
      </km-gcp-settings>
      <km-gke-settings *ngSwitchCase="Providers.GKE"
                       [formControl]="form.get(Controls.Settings)">
      </km-gke-settings>
      <km-hetzner-settings *ngSwitchCase="Providers.HETZNER"
                           [formControl]="form.get(Controls.Settings)">
      </km-hetzner-settings>
      <km-kubevirt-settings *ngSwitchCase="Providers.KUBEVIRT"
                            [formControl]="form.get(Controls.Settings)">
      </km-kubevirt-settings>
      <km-openstack-settings *ngSwitchCase="Providers.OPENSTACK"
                             [formControl]="form.get(Controls.Settings)">
      </km-openstack-settings>
      <km-equinix-settings *ngSwitchCase="Providers.EQUINIX"
                           [formControl]="form.get(Controls.Settings)">
      </km-equinix-settings>
      <km-nutanix-settings *ngSwitchCase="Providers.NUTANIX"
                           [formControl]="form.get(Controls.Settings)">
      </km-nutanix-settings>
      <km-vsphere-settings *ngSwitchCase="Providers.VSPHERE"
                           [formControl]="form.get(Controls.Settings)">
      </km-vsphere-settings>